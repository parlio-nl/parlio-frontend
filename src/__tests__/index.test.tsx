import { render, screen } from "@testing-library/react";
import Home from "../pages";

describe("/index", () => {
  it("renders a footer", () => {
    expect.assertions(2);

    render(<Home />);

    const footer = screen.getByTestId("home-footer");

    expect(footer).toBeInTheDocument();
    expect(footer).toHaveTextContent("Footer Text");
  });

  it("matches snapshot", () => {
    expect.assertions(1);

    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
