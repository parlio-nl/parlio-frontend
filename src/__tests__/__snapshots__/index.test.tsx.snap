// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/index matches snapshot 1`] = `
<div>
  <nav
    aria-label="breadcrumb"
    data-testid="breadcrumb-nav"
  >
    <ol
      class="breadcrumb"
    >
      <li
        class="breadcrumb-item"
      >
        <i
          class="bi bi-house-door-fill"
        />
         
        <a
          href="/"
        >
          Home
        </a>
      </li>
    </ol>
  </nav>
  <p>
    Test Text
  </p>
  <button
    class="btn btn-primary"
  >
    Test knop
  </button>
  <footer
    data-testid="home-footer"
  >
    <p>
      Footer Text
    </p>
  </footer>
</div>
`;
